tutorials/plot_visualize_evoked.py

```diff
@@ -115,6 +115,33 @@
                         ts_args=ts_args, topomap_args=topomap_args)
 
 ###############################################################################
+# Sometimes, you may want to compare two conditions at a selection of sensors,
+# or e.g. for the Global Field Power. For this, you can use the function
+# `mne.viz.plot_compare_evokeds`. The easiest way is to create a  Python
```

Comment:
Here you could use :func:\`mne.viz.plot_compare_evokeds\` for sphinx to link it.